pyscripts/s04_packages.py

```diff
@@ -115,63 +98,32 @@ def install_packages(user_input):
     package_list = misc_packages
     package_list += graphics_driver_packages[user_input['graphics driver']]
 
-    aur_package_list = []
-
     if 'full' in user_input['packages']:
         for _, value in packages.items():
             package_list += value[user_input['system type']]
         
-        for _, value in aur_packages.items():
-            aur_package_list += value[user_input['system type']]
-
-
         if user_input['desktop'] != 'none':
             package_list += desktop_distros[user_input['desktop']]
             for _, value in gui_packages.items():
                 package_list += value[user_input['system type']]
-            for _, value in aur_gui_packages.items():
-                aur_package_list += value[user_input['system type']]
 
     else:
         for package_type in set(['minimal'] + user_input['packages']):
             package_list += packages[package_type][user_input['system type']]
-            aur_package_list += aur_packages[package_type][user_input['system type']]
 
             if user_input['desktop'] != 'none':
                 package_list += desktop_distros[user_input['desktop']]
                 package_list += gui_packages[package_type][user_input['system type']]
-                aur_package_list += aur_gui_packages[package_type][user_input['system type']]
 
     package_string = " ".join(package_list)
-    aur_package_string = " ".join(aur_package_list)
-
-    with open('arch-installer/aur-packages/aurPackageList.txt', 'w') as txt_file:
-        txt_file.write(aur_package_string)
-
-
-
-    ### Nicer formatting for pacstrap
-    sed_inplace("/etc/pacman.conf", "#Color", "Color")
-    sed_inplace("/etc/pacman.conf", "#TotalDownload", "TotalDownload")
-
-    ### Update mirrorlist
-    print(" >> Updating mirror list")
-    try:
-        run('reflector --latest 100 --sort rate --protocol https --save /etc/pacman.d/mirrorlist')
-    except CalledProcessError as error:
-        print('Updating package mirrorlist failed with message: ', error.output)
-
-
-
-    print(" >> Going to install arch packages")
-    try:
-        run('pacstrap /mnt '+ package_string)
-    except CalledProcessError as error:
-        print('Error installing packages. Message: ', error.output)
-
-    print(" >> Installed arch packages")
 
+    print(" >> Going to install user packages")
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.txt', dir='/mnt') as package_file:
+        package_file.write(package_string)
+        package_file.flush()
+        try:
+            run('arch-chroot /mnt sudo -u {} pacaur -S --noconfirm --noedit `cat {}`'.format(install_user_name, package_file.name))
```

Comment:
Why with a file using `cat` instead of appending the `package_string` directly to the string passed to `run`?